src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs

```diff
@@ -56,6 +57,13 @@ private void _vm_PropertyChanged(object sender, System.ComponentModel.PropertyCh
                 else
                 {
                     this.Height = minHeightSize;
+                    if (_vm.Children.Count !=0)
+                    {
+                        if (_vm.Children[0].NodeLabel.Contains(Environment.NewLine))
+                        {
+                            this.Height = defaultHeightSize;
```

Comment:
In which case is `defaultHeightSize` still used? Should we count the number of `Environment.NewLine` and then multiply `minHeightSize` to be more accurate?

Reply:
defaultHeightSize is used when the node display a collection / the second question depends on the Business, I guess Aabishkar KC could help us.

Reply:
@Amoursol  or @Jingyi-Wen  would know better

Reply:
@jesusalvino Then is minHeight including the watch node header? If yes, maybe this is good enough for now